docs/api-react/schematics/application.md

```diff
@@ -11,6 +11,14 @@ ng generate application ...
 
 ## Options
 
+### babel
+
+Default: `false`
```

Comment:
If babel is a default for most folks, should we make it a default or you think it is too risky?

Reply:
@FrozenPandaz and I discussed it and the worry is that libs who use advanced babel features (like macros) would not be usable within apps that use ts-loader. 

Currently if the workspace is created as react workspace, the app generates using `babel: true`, otherwise it's false to keep compatibility easier. We can give this more thought.